### DISCUSSION TITLE
CORS issues

### DISCUSSION THREAD

**Discussion by Ilya-Kaplan-Alphaguilty** [WalletConnect/walletconnect-monorepo#3853] [Developer Support] [answered]
Hi I have an issue with CORS errors from Ukraine, with VPN everything works fine

<img width="767" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/125646168/2fbbd5ad-714a-4e91-ab0f-954e36db64de">
<img width="789" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/125646168/0d399690-35f6-4aad-ae45-426f4f84f047">

<img width="767" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/125646168/b2137d5e-d371-4628-9977-0141eadbf949">
<img width="789" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/125646168/3bdf5e50-3e0e-437d-ac05-4bf93747db03">

**Comment by gymnasy55** [comment]
Same here https://github.com/orgs/WalletConnect/discussions/3842

**Comment by glitch-txs** [comment, marked as answer]
Please read here
https://github.com/orgs/WalletConnect/discussions/3862
